app/assets/locales/android_translatable_strings.txt

```diff
@@ -529,6 +529,10 @@ login.attempt.fail.changed.title=Password Changed since Last Login
 login.attempt.fail.changed.detail=You've logged in with a new password since you're last login. You may need to log in with your old credentials to ensure you have no unsent data.
 login.attempt.fail.changed.action=If you had unsent data or incomplete forms on the phone before you logged in with your new password you should log out, and log in with your old password. Make sure all your unsent forms are sent and incomplete forms are submitted, then log in again with your new password and sync. 
 
+login.attempt.insufficient.role.permission.title=User doesn't have permission to access the CommCare application
+login.attempt.insufficient.role.permission.detail=The role assigned to your user doesn't have permission to access CommCare mobile application
+login.attempt.insufficient.role.permission.action=Contact your CommCare Administrator or CommCare Support for assistance with this matter

```

Comment:
`CommCare Administrator or CommCare Support`

lets just say `Support` here and let it mean whatever user associate with support.

Reply:
[8ba5745](https://github.com/dimagi/commcare-android/pull/2975/commits/8ba57452f22aece078432bc5933c450b6c8c04b4)